.github/workflows/playwright.yml

```
@@ -0,0 +1,50 @@
+      with:
+        node-version: lts/*
+
+    - name: Install dependencies
+      run: npm install -g yarn && yarn
+
+    - name: Install Playwright Browsers
+      run: yarn playwright install --with-deps
+
+    - name: Run Playwright tests
+      run: yarn playwright test
+      env: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)